pkg/streamingpromql/query.go

```diff
@@ -271,26 +271,35 @@ func (q *Query) populateVectorFromInstantVectorOperator(ctx context.Context, o o
 			return nil, err
 		}
 
-		if len(d.Floats)+len(d.Histograms) != 1 {
-			operator.PutFPointSlice(d.Floats)
-			// TODO: put histogram point slice back in pool
-
-			if len(d.Floats)+len(d.Histograms) == 0 {
-				continue
-			}
+		defer operator.PutFPointSlice(d.Floats)
+		defer operator.PutHPointSlice(d.Histograms)
```

Comment:
Given `PutFPointSlice` and `PutHPointSlice` should always be used at the same time like this for `InstantVectorSeriesData` values, what do you think about adding a helper method like `PutInstantVectorSeriesDataSlices`?

Reply:
I think that makes sense. I might make it local to this function to begin with though.